test.sh

```diff
@@ -48,20 +61,14 @@ for dir in ${VERSIONS}; do
   fi
 
   if [ -n "${TEST_OPENSHIFT_4}" ]; then
+    export IMAGE_NAME="${NAMESPACE}${BASE_IMAGE_NAME}-${dir//./}-${OS}"
```

Comment:
Is this necessary? `IMAGE_NAME` is set a few lines above, at the start of the cycle.